blaze/compute/tests/test_sql_compute.py

```diff
@@ -556,6 +556,16 @@ def test_relabel_table():
     assert str(result) == str(expected)
 
 
+def test_relabel_columns_over_selection():
+    result = compute(t[t['amount'] > 150].relabel(name='NAME'), s, return_type
+    = 'native')
+    expected = str(sa.select(
+        [s.c.name.label('NAME'), s.c.amount, s.c.id]
+    ).where(s.c.amount > 150))
+    import pdb; pdb.set_trace()
+    assert str(result) == str(expected)
```

Comment:
Whoops - thought this was a comment for `test_postgresql_compute`. Makes more sense now.